qdrant-landing/content/documentation/hybrid-cloud/platform-deployment-options.md

```diff
@@ -5,34 +5,136 @@ weight: 4
 
 # Platform Deployment Options
 
-## Amazon Web Services
+![Akamai](/documentation/cloud/cloud-providers/akamai.jpg)
 
+## Akamai (Linode)
 
-## Google Cloud Platform
+[The Linode Kubernetes Engine (LKE)](https://www.linode.com/products/kubernetes/) is a managed container orchestration engine built on top of Kubernetes. LKE enables you to quickly deploy and manage your containerized applications without needing to build (and maintain) your own Kubernetes cluster. All LKE instances are equipped with a fully managed control plane at no additional cost.
 
+### More on Linode Kubernetes Engine
 
-## Microsoft Azure
+- [Getting Started with LKE](https://www.linode.com/docs/products/compute/kubernetes/get-started/)
+- [LKE Guides](https://www.linode.com/docs/products/compute/kubernetes/guides/)
+- [LKE API Reference](https://www.linode.com/docs/api/)
 
+![AWS](/documentation/cloud/cloud-providers/aws.jpg)
 
-## DigitalOcean
+## Amazon Web Services (AWS)
 
+[Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/) is a managed service to run Kubernetes in the AWS cloud and on-premises data centers which can then be paired with Qdrant's hybrid cloud. With Amazon EKS, you can take advantage of all the performance, scale, reliability, and availability of AWS infrastructure, as well as integrations with AWS networking and security services.
+
+### More on Amazon Elastic Kubernetes Service
```

Comment:
EKS does not automatically install CSI and Snapshot controller. You have to add this as an add-on

https://docs.aws.amazon.com/eks/latest/userguide/managing-ebs-csi.html
https://docs.aws.amazon.com/eks/latest/userguide/csi-snapshot-controller.html

Reply:
You also need to create a storageclass manually with allowVolumeExpansion set to true, and a corresponding VolumeSnapshotClass